### PR TITLE
Prl 660 labelling mandatory fields

### PR DESCRIPTION
**Pull request by khurshedk**
**Before creating a pull request make sure that:**

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-863

### Change description ###

* added to mandatory fields

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: definitions/private-law/json/CaseEventToFields/OtherPeopleInTheCase/CaseEventToFields.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "DisplayContext": "COMPLEX",
     "PageID": 1,
     "PageDisplayOrder": 1,
-    "PageFieldDisplayOrder": 1,
+    "PageFieldDisplayOrder": 2,
```
Comment (AlokDatta) [inline definitions/private-law/json/CaseEventToFields/OtherPeopleInTheCase/CaseEventToFields.json:10]: Do we need this change in display order?

File: definitions/private-law/json/ComplexTypes/PartyDetails/ComplexTypes.json [hunk 2]
```diff
@@ -162,71 +162,71 @@
...
+    "ElementLabel": "*Do you need to keep the contact number confidential?",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "05/11/2021",
     "ID": "PartyDetails",
     "ListElementCode": "addNewApplicantLabel",
     "FieldType": "Label",
-    "ElementLabel": "## Add new applicant",
+    "ElementLabel": "Add new applicant",
```
Comment (AlokDatta) [inline definitions/private-law/json/ComplexTypes/PartyDetails/ComplexTypes.json:181]: check this one too?

File: definitions/private-law/json/ComplexTypes/PartyDetails/ComplexTypes.json [hunk 3]
```diff
@@ -341,7 +341,7 @@
     "ID": "PartyDetails",
     "ListElementCode": "addNewRespondentLabel",
     "FieldType": "Label",
-    "ElementLabel": "## Add new respondent",
+    "ElementLabel": "Add new respondent",
```
Comment (AlokDatta) [inline definitions/private-law/json/ComplexTypes/PartyDetails/ComplexTypes.json:344]: check this one

**Review by AlokDatta — CHANGES_REQUESTED** [review]

File: definitions/private-law/json/CaseEventToFields/OtherPeopleInTheCase/CaseEventToFields.json (hunk 1 above)
Comment (khurshedk) [reply definitions/private-law/json/CaseEventToFields/OtherPeopleInTheCase/CaseEventToFields.json:20]: there were field added as part of PRL-660, so added that again

File: definitions/private-law/json/ComplexTypes/PartyDetails/ComplexTypes.json (hunk 2 above)
Comment (khurshedk) [reply definitions/private-law/json/ComplexTypes/PartyDetails/ComplexTypes.json]: replaced with version from master

File: definitions/private-law/json/ComplexTypes/PartyDetails/ComplexTypes.json (hunk 3 above)
Comment (khurshedk) [reply definitions/private-law/json/ComplexTypes/PartyDetails/ComplexTypes.json]: replaced text from master.

**Comment by AlokDatta** [comment]
Closing, will be merged as part of https://github.com/hmcts/prl-ccd-definitions/pull/247

**Pull request closed without merging**